rust/libnewsboat/src/utils.rs

```diff
@@ -110,6 +110,15 @@ pub fn resolve_relative(reference: &Path, path: &Path) -> PathBuf {
 	return path.to_path_buf();
 }
 
+#[test]
+fn t_resolve_relative() {
```

Comment:
Please move this further down, into `tests` module. That module is marked with `#[cfg(test)]`, meaning the tests won't even be compiled if there is no intention to run them.